starter-kits.json

```diff
@@ -4,6 +4,7 @@
   "cra-rxjs-styled-components": "Create React App, RxJS and Styled Components",
   "deno-oak-denodb": "Deno, Oak, and DenoDB",
   "expo-zustand-styled-component": "Expo, Zustand, and Styled Components",
+  "express-apollo-prisma": "Express, Apollo and Prisma",
```

Comment:
```suggestion
  "express-apollo-prisma": "Express.js, Apollo Server, and Prisma",
```

Reply:
Not because I need it to be those exact texts, but it's more inline with what we have in the other descriptions ;)